scripts/core/registry.yml

```diff
@@ -400,18 +400,12 @@ etors:
 - name: BINDLESS_IMAGES_MIPMAP_FREE_EXP
   desc: Enumerator for $xBindlessImagesMipmapFreeExp
   value: '142'
-- name: BINDLESS_IMAGES_IMPORT_OPAQUE_FD_EXP
-  desc: Enumerator for $xBindlessImagesImportOpaqueFDExp
-  value: '143'
 - name: BINDLESS_IMAGES_MAP_EXTERNAL_ARRAY_EXP
   desc: Enumerator for $xBindlessImagesMapExternalArrayExp
   value: '144'
 - name: BINDLESS_IMAGES_RELEASE_INTEROP_EXP
   desc: Enumerator for $xBindlessImagesReleaseInteropExp
   value: '145'
-- name: BINDLESS_IMAGES_IMPORT_EXTERNAL_SEMAPHORE_OPAQUE_FD_EXP
```

Comment:
is there a reason we can't re-use the old enum number when something gets renamed?

Reply:
They are supposed to be globally unique for xpit I think.